src/components/FeedbackModal.tsx

```diff
@@ -20,7 +19,7 @@ interface FeedbackModalProps {
   feedbackForId: number
 }
 
-const FeedbackModal: React.FC<FeedbackModalProps> = ({
+const FeedbackModal: FC<FeedbackModalProps> = ({
```

Comment:
For consistency reason i think we should remove the type here since we never used in other components
`: FC<FeedbackModalProps>` is not needed